www/src/pages/blog/astro-021-preview.astro

```diff
@@ -34,7 +34,7 @@ let lang = 'en';
         - [Brand new features, like Components-in-Markdown](#components-in-markdown)
         - [A new system for HTML live-updating via HMR](#hmr-meet-html)
 
-        You can try out our latest release today [in the browser](https://gitpod.io#snapshot/5e7cf2f1-8108-4fa5-99d3-ed8de70d8c23) or by running `npm install astro@next--compiler` in a new project directory on your machine.
```

Comment:
You might want to update the gitpod url as well - Maybe https://gitpod.io/#https://github.com/snowpackjs/astro/

Reply:
Will do, thanks!